repos/system_upgrade/el7toel8/actors/dhclientreport/actor.py

```diff
@@ -0,0 +1,37 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckDhclientActor(Actor):
+    """
+    Check if dhclient is installed
+    and report backward incompatible change in CLI options
+    """
+
+    name = "dhclient"
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def is_dhclient_installed():
```

Comment:
Missing self argument
```suggestion
    def is_dhclient_installed(self):
```